google/cloud/aiplatform/experimental/vertex_model/utils/source_utils.py

```diff
@@ -87,28 +87,36 @@ def _make_source(
     src = src + "if __name__ == '__main__':\n"
 
     # Then, instantiate model
-    src = src + f"\tmodel = {cls_name}()\n"
+    # First, grab args and kwargs using the _constructor_arguments variable in VertexModel
+    class_args = inspect.signature(obj.__class__.__init__).bind(
+        obj, *obj._constructor_arguments[0], **obj._constructor_arguments[1]
+    )
+
+    # need to index pass the first arg to avoid the call to self.
+    src = src + f"\tmodel = {cls_name}({class_args.args[1:]}, {class_args.kwargs})\n"
 
-    src = src + f"\tmodel.{instance_method}("
+    if instance_method is not None:
```

Comment:
When would instance_method be `None`? It seems like it's required based on the signature of `_make_source`.

Reply:
I used `None` during testing to test that the multiple constructor arguments passed.